tutorials-v5/heom/heom-6-parity.md

```diff
@@ -0,0 +1,244 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python
+import numpy as np
+import matplotlib.pyplot as plt
+from qutip import tensor,destroy,qeye,spre,spost,operator_to_vector,expect,qeye
+from qutip.solver.heom import FermionicBath,LorentzianPadeBath,HEOMSolver
+import scipy.sparse as sp
+from scipy.sparse.linalg import lgmres
+```
+
+<!-- #region -->
+# Kondo  Peak
+
+In this tutorial we will find the kondo peak for a system of two spins connected to two fermionic reservoirs 
+
+The Hamiltonian of this setup is given by:
+
+$H_{T}=H_{S}+H_{f}+H_{ef}$
+
+Where $H_{S}$ is the system Hamiltonian, which is divided into three contributions. The electron system hamiltonian ($H_{e}$), the single-mode cavity ($H_{c}$) and the light matter coupling ($H_{ef}$):
```

Comment:
there is no cavity in this example, better to remove mention of it here and elsewhere.